src/routes/sequencing/new/+page.ts

```diff
@@ -3,9 +3,9 @@ import { redirect } from '@sveltejs/kit';
 import type { PageLoad } from './$types';
 
 export const load: PageLoad = async ({ parent }) => {
-  const { user } = await parent();
+  const { user, permissibleQueries } = await parent();
 
-  if (!user) {
+  if (!user || (permissibleQueries && !Object.keys(permissibleQueries).length)) {
```

Comment:
Thoughts on making this a util? Downside is it would obscure what happens here but upside is that there'd be a little less boilerplate logic for each page?

Reply:
Hmmm that's a good point. Do you mind if I address this in a follow-up PR? I have some changes in another branch that touch this and I'd like to avoid a conflict if possible.